examples/plotting/200_cell_transitions.ipynb

```diff
@@ -101,38 +101,11 @@
      "name": "stdout",
```

Comment:
<p>now you need to say which solution of the <code>cell_transition</code> you pass. i.e. whether you set <code>key_added</code> etc. because last <code>cell_transitino</code> you did is not the one you're plotting.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>